ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -149,6 +152,7 @@ public void setAccessibilityLiveRegion(T view, String liveRegion) {
     }
   }
 
+  private static final float MatrixPerspectiveToCameraDistanceMultiplier = 5;
```

Comment:
Nit: This should probably be written LIKE_THE_OTHER_CONSTANTS and put at the top of the file.